### DISCUSSION TITLE
Zigbee Dual Meter (https://es.aliexpress.com/item/1005005787993771.html?gatewayAdapt=glo2esp)

### DISCUSSION THREAD

**Discussion by Waterbrain** [Koenkk/zigbee2mqtt#18662] [Devices]
Hello:
I wonder if the integration of this device is planned.
If not, I could develop the support.
But I would need some instructions since in my investigations I have not found anything that clarifies the process for me.
Thanks and best regards

**Comment by netweaver1970** [comment]
when you create a new issue in Github, specifically for a new device support request, you get this:

![image](https://github.com/Koenkk/zigbee2mqtt/assets/18028618/ee748551-91e3-4740-96af-01149c23b901)

and there is this link: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

it kind of works, you can start the legwork for Koen and if you run into questions/problems, you can raise an issue and ask for help. Be prepared for lots of reading/searching etc for your 1st device. I went through it this week and I still hardly know exactly how it all fits together. But at least I have something working. Based on the guide.

**Comment by Waterbrain** [comment]
Hello:

I'm trying to follow the guide https://www.zigbee2mqtt.io/advanced/support-new-devices/03_find_tuya_data_points.html#_8-find-your-data-point <https://www.zigbee2mqtt.io/advanced/support-new-devices/03_find_tuya_data_points.html#_8-find-your-data-point>

But i have a problem…….



As I could see when executing export_codes()



I would appreciate some kind of help.

Take care and enjoy the rest of your day.

Best regards,

Antonio Maza Bandrés
***@***.***
